Auth0.Windows/Auth0Client.cs

```diff
@@ -199,10 +205,20 @@ public Task<JObject> GetDelegationToken(string targetClientId, IDictionary<strin
         /// <summary>
         /// Log a user out of a Auth0 application.
         /// </summary>
-        public void Logout()
+        /// <param name="forceClearCache">Force clearing the browser cache, no matter the login method</param>
+        public void Logout(bool forceClearCache = false)

```

Comment:
Yes I understand. Let me take it up with the original developer on our side to understand the reason for this. Let's just hang on with this for a day or two until I have cleared it up with him.